build/webpack/common.js

```diff
@@ -6,15 +6,7 @@ const glob = require('glob');
 const path = require('path');
 const webpack_bundle_analyzer = require('webpack-bundle-analyzer');
 const constants = require('../constants');
-exports.nodeModulesToExternalize = [
-    '@jupyterlab/services',
-    'node-stream-zip',
```

Comment:
These are either inlined today, hence marking them as externals is a moot point (just adding more into a seprate bundle).